tests/test_cmd_resetdep.py

```diff
@@ -0,0 +1,98 @@
+import pytest
+from six import StringIO
+
+from doit.cmd_resetdep import ResetDep
+from doit.dependency import TimestampChecker, get_md5, get_file_md5
+from doit.exceptions import InvalidCommand
+from doit.task import Task
+from tests.conftest import tasks_sample, CmdFactory, get_abspath
+
+
+class TestCmdResetDep(object):
+
+    def test_execute(self, depfile, dependency1):
+        output = StringIO()
+        tasks = tasks_sample()
+        cmd_list = CmdFactory(ResetDep, outstream=output, task_list=tasks,
```

Comment:
I guess you used test_cmd_list as a base for this file :)
Please replace `cmd_list` with `cmd_reset`...